v2/refs/marshal.go

```diff
@@ -123,6 +133,23 @@ func (o *ObjectID) StableSize() int {
 	return proto.BytesSize(objectIDValField, o.val)
 }
 
+// ObjectIDNestedListSize returns byte length of nested
```

Comment:
Fix comment, it is from `ObjectIDNestedListSize()`.

Reply:
Fixed